src/backends.jl

```diff
@@ -289,7 +286,7 @@ end
 # ------------------------------------------------------------------------------
 # gr
 
-_initialize_backend(pkg::GRBackend) = nothing  # COV_EXCL_LINE
+_initialize_backend(pkg::GRBackend) = @eval Plots import GR  # COV_EXCL_LINE
```

Comment:
```suggestion
_initialize_backend(pkg::GRBackend) = nothing  # COV_EXCL_LINE
```

Since `GR` is still imported in `backends/gr.jl`, I don't think this is needed.